client/lib/post-normalizer/index.js

```diff
@@ -509,6 +511,7 @@ normalizePost.content = {
 					// fun fact: removing the node from the DOM will not prevent it from loading. You actually have to
 					// change out the src to change what loads. The following is a 1x1 transparent gif as a data URL
 					image.setAttribute( 'src', 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7' );
+					image.removeAttribute( 'srcset' );
```

Comment:
Does `srcset` really load if not inserted into the DOM? Better safe than sorry, but now you've got me curious :smile: